us/en/skymiles/blocks/blade/blade.js

```diff
@@ -6,13 +6,14 @@ export default function decorate(block) {
       } else {
         div.className = 'blade-body';
         const a = div.lastElementChild.querySelector('a:only-child:last-child');
+        a.classList.add('wrap');
         try {
           if (a && new URL(a.href).pathname === new URL(a.textContent).pathname) {
             div.parentElement.classList.add('is-link');
-            div.addEventListener('click', () => {
-              window.location.href = div.querySelector('a').href;
-            });
-            a.parentElement.style.display = 'none';
+            a.innerHTML = row.innerHTML;
+            a.querySelector('a').remove();
+            row.innerHTML = a.outerHTML;
```

Comment:
Just realized over the weekend that `outerHTML` is a performance killer. We are better off doing:
```
a.innerHTML = '';
a.append(row);
```